index.d.ts

```diff
@@ -4,20 +4,36 @@ declare module "react-date-picker" {
   export default function DatePicker(props: DatePickerProps): JSX.Element;
 
   export interface DatePickerProps extends CalendarProps {
+    autoFocus?: boolean;
+    calendarAriaLabel?: string;
     calendarClassName?: string | string[];
-    calendarIcon?: React.ReactElement;
-    className?: string | string[];
+    calendarIcon?: string | React.ReactNode;
```

Comment:
Can you tell me what was wrong here? Isn't string a valid React Node?

Reply:
That was a mistake on my part. I changed JSX.Element to React.ReactNode and forgot to remove the string type.